assets/pages/new-contributions-landing/components/ButtonWithRightArrow.jsx

```diff
@@ -17,20 +17,21 @@ type PropTypes = {
 // ----- Render ----- //
 
 function ButtonWithRightArrow(props: PropTypes) {
+  const buttonElem = (
+    <button
+      className={props.buttonClassName}
+      type={props.type}
+      aria-describedby={props.accessibilityHintId}
+      onClick={props.onClick}
+    >
+      {props.buttonCopy}
+      <p id={props.accessibilityHintId} className="accessibility-hint">{props.buttonCopy}</p>
+      <SvgArrowRight />
+    </button>);
 
   return (
     <div className={props.componentClassName}>
-      <button
-        className={props.buttonClassName}
-        type={props.type}
-        href={props.url}
-        aria-describedby={props.accessibilityHintId}
-        onClick={props.onClick}
-      >
-        {props.buttonCopy}
-        <p id={props.accessibilityHintId} className="accessibility-hint">{props.buttonCopy}</p>
-        <SvgArrowRight />
-      </button>
+      { props.url ? <a href={props.url}>{buttonElem}</a> : buttonElem }
```

Comment:
I see what you are doing here, however this is problematic for UAs: `<a><button>...`. I would instead choose either `A` or `BUTTON` and stick to it.